internal/model/label.go

```diff
@@ -0,0 +1,14 @@
+package model
+
+// Label represents a label shown in the Marketplace UI.
+type Label struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	URL         string `json:"url"`
+}
+
+var OfficialLabel Label = Label{
+	Name:        "Official",
+	Description: "This plugin is maintained by Mattermost",
```

Comment:
@aaronrothschild Just want to double check that you are fine with this wording.

Reply:
Yes, I think that's accurate.

Reply:
Should we trail with a period, or do we not do that with the labels today?

Reply:
I'm 0/5.

Reply:
@aaronrothschild What do you think about the Jesse's proposal?

Reply:
no trailing dot is good

Reply:
:+1: I changed this also for the `Local` label in https://github.com/mattermost/mattermost-server/pull/13425/commits/018fd8d6961f0fee06df69f8aa394b0227305d83